tck/utils/common-tx-params.ts

```diff
@@ -0,0 +1,56 @@
+import {
+    Client,
+    Transaction,
+    Hbar,
+    TransactionId,
+    PrivateKey,
+} from "@hashgraph/sdk";
+
+import { ApplyCommonTransactionInputParams } from "../models/common-tx-params";
+import { getKeyFromString } from "../methods/key";
+
+export const applyCommonTransactionParams = (
```

Comment:
Can this be moved inside `common-tx-params.ts` in the /params folder alongside with the interface. I think it looks cleaner when the logic is in the same file.
nit: it make sense for `ApplyCommonTransactionInputParams` to be a class, but this still works i guess.